aws-java-sdk-dynamodb/src/main/java/com/amazonaws/services/dynamodbv2/datamodeling/ConversionSchemas.java

```diff
@@ -158,6 +158,89 @@
 
     static final ConversionSchema DEFAULT = V2_COMPATIBLE;
 
+    /**
+     * A ConversionSchema builder that defaults to the {@link #V1} schema.
+     */
+    public static final Builder v1Builder(String name) {
```

Comment:
Is it possible to define the constants above via the builder? Also final is redundant for static methods.